theoplayer/how-to-guides/android/media3/getting-started.mdx

```diff
@@ -0,0 +1,99 @@
+# Getting started with Media3 on Android
+
+<Intro />
```

Comment:
This component doesn't exist. (I assume you copied this from #143, where it's an imported component to have the same introduction across Web and Android.)

Remove it, and write your own introduction inline. 😉